sktime/classification/ensemble/_ctsf.py

```diff
@@ -188,7 +203,7 @@ class labels (multi-output problem).
 
     def __init__(
         self,
-        estimator=None,
+        estimator,
```

Comment:
I am not sure if this is acceptable in a example addition PR. Plus it's potentially a breaking change as it now wants makes estimator a positional argument from a keyword argument. In fact this is what causes the test failure in CI

FYI @fkiraly